packages/@aws-cdk/aws-elasticloadbalancingv2/lib/nlb/network-load-balancer.ts

```diff
@@ -38,6 +38,14 @@ export interface NetworkLoadBalancerAttributes {
    * @default - When not provided, LB cannot be used as Route53 Alias target.
    */
   readonly loadBalancerDnsName?: string;
+
+  /**
+   * The optional VPC to associate with the imported load balancer.
```

Comment:
nit: I'd refrain from describing this as optional, and just mention `The VPC to associate to the imported load balancer`